dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -591,9 +602,6 @@ def process_data_chunk(self, chunk_df, shared_data, m_lock=None):
         # index gets reset for each chunk, adding the header, and adding previous rows
         chunk_df['row_number'] = chunk_df.index + 2
 
-        # Some long null rows may get included in the resulting dataframe, this ensures they are removed
-        chunk_df = chunk_df[~chunk_df['row_number'].isin(self.long_null_rows)]
-
```

Comment:
Is it no longer possible for a chunk to have long rows? It can't have long rows at the beginning but what if a chunk has a long row at the end? Or in the middle? We still don't want those right?

Reply:
They still shouldn't get surfaced via pandas like normal. They only randomly surface when the whole chunk is filled with rows with format errors. The reason why we skip them in the front is that pandas processes the rest of the rows wrong if it tries to process the first long rows.